app/adapters/external/cache/infinite_buying_adapter.py

```
@@ -630,29 +439,27 @@
 
     async def close(self) -> None:
-        """연결을 종료합니다."""
+        """Cache 연결을 종료합니다."""
         if self._client:
             try:
                 await self._client.close()
-            except ClosingError as e:
-                logger.error(f"Cache 연결 종료 중 오류: {e}")
-            else:
                 logger.info("Cache 연결이 종료되었습니다.")
+            except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)